### PR TITLE
fix(Core/GameObjectScript): Load ICC Scourge Teleport gossip from DB

### PR DESCRIPTION
**Pull request by brussens**
##### CHANGES PROPOSED:

ICC Scourge teleport load gossip options from database.

##### TESTS PERFORMED:

Tested on Win10 PRO x64/Debian 9 x64

##### HOW TO TEST THE CHANGES:

Open Scourge Teleporter in ICC and you shouldn't see any changes.

##### Target branch(es):

Master

 
 

 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

### CODE REVIEW

**Comment by brussens** [comment]
@Stoabrogga no problem

**Comment by Stoabrogga** [comment]
@brussens 
Great, thanks.

File: src/server/scripts/Northrend/IcecrownCitadel/icecrown_citadel_teleport.cpp [hunk 1]
```diff
@@ -21,20 +31,38 @@ class icecrown_citadel_teleport : public GameObjectScript
             uint32 gossipMenuId = go->GetGOInfo()->GetGossipMenuId();
 
             if (go->GetEntry() != GO_SCOURGE_TRANSPORTER_FIRST)
-                AddGossipItemFor(player, gossipMenuId, 0, GOSSIP_SENDER_ICC_PORT, LIGHT_S_HAMMER_TELEPORT); // M_PI + M_PI/6
+                AddGossipItemFor(
+                    player, gossipMenuId, ICC_TELEPORT_GOSSIP_OPT_LIGHTS_HAMMER,
+                    GOSSIP_SENDER_ICC_PORT, LIGHT_S_HAMMER_TELEPORT
+                ); // M_PI + M_PI/6
```
[the remark this answers is not in the document]
Comment (FrancescoBorzi) [reply src/server/scripts/Northrend/IcecrownCitadel/icecrown_citadel_teleport.cpp]: when line gets too long, I agree to split in parameters like this. However we should agree on when it's considered "too long"

**Comment by Stoabrogga** [comment]
Tested this successfully, the texts where the same.

**Pull request merged**